### ISSUE TITLE
TestSet not loading any sample

### ISSUE THREAD

**Issue by LuigiSigillo** [2165998/TarGAN#2]
Hi, I tried your network, but I have some problems in the loading of the dataset, for instance the testset.
I noticed that the testset, downloaded from the chaos website, has no ground truth, so there is no folder named "ground". Therefore when you try to load the  ChaosDataset_Syn_Test it will say: 
`modal:t1,fold:test/t1, total size:0`
Is this a problem with the structure of the folder, or I have to split the train set that contains also the folder ground manually and not use at all the test set provided by chaos?

**Comment by cs-xiao** [comment]
Hi, you can split the train set from CHAOS into training set and test set.
We just use the training data from CHAOS in our experiments.

**Comment by LuigiSigillo** [comment]
Yes, that's what I have already done, thank you.
